File: Java/Somatorio.java
public class Somatorio {

    public static double somatorio(double[] numeros) {
        double soma = 0;

        for (int i = 0; i < numeros.length; i++) {
            soma += numeros[i];
        }
        return soma;
    }

    public static void main(String[] args) {
        double[] numeros = {1, 2, 3, 4, 5};

        System.out.println("--- Somatorio ---");
        System.out.println("Quantidade de numeros: " + numeros.length);
        System.out.println("Soma: " + somatorio(numeros));
    }
}

File: JavaScript/somatorio.js
function Somatorio(numeros) {
  let soma = 0;

  for (let i = 0; i < numeros.length; i++) {
    soma += numeros[i];
  }
  return soma;
}

const numeros = [1, 2, 3, 4, 5];

console.log("--- Somatório ---");
console.log("Números: " + numeros.join(", "));
console.log("Soma: " + Somatorio(numeros));